prov/efa/src/rxr/rxr_pkt_type_req.c

```diff
@@ -852,10 +968,21 @@ ssize_t rxr_pkt_proc_matched_medium_rtm(struct rxr_ep *ep,
 
 	if (rx_entry->total_len == rx_entry->bytes_done) {
 		rxr_pkt_rx_map_remove(ep, pkt_entry, rx_entry);
-		/*
-		 * rxr_cq_handle_rx_completion() releases pkt_entry, thus
-		 * we do not release it here.
-		 */
+		if (pkt_type == RXR_DC_MEDIUM_MSGRTM_PKT || pkt_type == RXR_DC_MEDIUM_TAGRTM_PKT) {
```

Comment:
Looks like `RXR_REQ_WAIT_RECEIPT` was added for medium message but this wasn't updated.

Reply:
Currently we are handing the receipt packet sending logic in `rxr_pkt_handle_data_copied`. The `RXR_REQ_WAIT_RECEIPT `flag is no longer used.